source/jormungandr/jormungandr/scenarios/journey_filter.py

```diff
@@ -337,6 +337,8 @@ def filter_func(self, journey):
             return False
         elif self.dp == 'only' and 'non_pt' not in journey.tags:
             return False
+        elif self.dp == 'only_with_alternatives' and 'non_pt' not in journey.tags:
+            return False
```

Comment:
Does it make sense to compress these four lines into 2?
```suggestion
        elif self.dp in ('only', 'only_with_alternatives') and 'non_pt' not in journey.tags:
            return False
```

Reply:
Yes you're right, it makes more sense !